media_driver/linux/common/codec/ddi/media_ddi_decode_const.h

```diff
@@ -41,4 +41,25 @@
 #define DECODE_ID_JPEG      "VIDEO_DEC_JPEG"
 #define DECODE_ID_VC1       "VIDEO_DEC_VC1"
 
+#define AVC_DECODE_MAX_WIDTH          8192
```

Comment:
1. I think need more comments for the MAX size
2. Do you lost VP8?

Reply:
VP8 used DEFAULT_DECODE_MAX_WIDTH and DEFAULT_DECODE_MAX_Height